extensions/vscode/src/granite/panels/setupGranitePage.ts

```diff
@@ -396,7 +396,7 @@ export class SetupGranitePage {
       if (result) {
         await this.saveSettings(modelSize);
         this.hideGraniteOnboardingCard();
-
+        commands.executeCommand("continue.continueGUIView.focus");
```

Comment:
Yeah, I think that's better. It's less confusing if it doesn't just open when the user does Cmd-L/Cmd-I

Reply:
actually it's original code from paver that didn't make it to granite.code. Simply restoring things the way they should be ;-)